src/qt/rpcconsole.cpp

```diff
@@ -706,6 +706,14 @@ void RPCConsole::setClientModel(ClientModel *model, int bestblock_height, int64_
 
         // create ban table context menu
         banTableContextMenu = new QMenu(this);
+        /*: Context menu action to copy the IP/Netmask of a banned peer.
+            IP/Netmask is the combination of a peer's IP address and it's Netmask.
+            For IP Address see: https://en.wikipedia.org/wiki/Subnetwork
```

Comment:
This is similar to what was done in #332, see: https://github.com/bitcoin-core/gui/pull/332/files#diff-fe41db46679f280131a7b0a04d39383dd4d0ab623cec8de8ddc74f79b163dabfR285

But, is this an appropriate link for more information on `IP Address`. What about: https://en.wikipedia.org/wiki/IP_address